cartography/intel/helper/google_request.py

```diff
@@ -0,0 +1,33 @@
+import logging
+import time
+
+from googleapiclient.discovery import HttpError
+logger = logging.getLogger(__name__)
+
+
+def repeat_request(req, req_args, req_next=None, retries=5, retry_delay_ms=500):
```

Comment:
Could we implement this as generic retry operation we can use across all modules and tasks?

For example of method signature:
invoke_with_retry(operation, retries, retry_delay, retriable_exception_types[])